cli/enginecli/test_dropper.py

```diff
@@ -90,6 +90,10 @@ def setUpClass(cls) -> None:
             cls.mintable_terminus_pool_id, cls.dropper.address, {"from": accounts[0]}
         )
 
+        terminus_info_1, terminus_info_2 = cls.dropper.get_terminus_admin_info()
+        DropperTestCase().assertEqual(cls.terminus.address, terminus_info_1)
```

Comment:
@ShivaLaguna : Can you make a separate test for this? I don't like to put tests in the setup code.